### ISSUE TITLE
Is there a way to run as "go test -parallel 1"?

### ISSUE THREAD

**Issue by vsayer** [smartystreets/goconvey#444]
I have a battery of tests which hit postgresql and is not designed to run concurrently yet. The concurrency is being worked on, but in the meantime is there a way to run tests sequentially?

**Comment by riannucci** [comment]
Is this multiple test binaries that are hitting postgres, or one test binary with a whole bunch of test functions in it? I believe that if it's just the one binary, and the test functions don't call t.Parallel(), they'll run sequentially by default.

**Comment by vsayer** [comment]
Nope, unfortunately the tests are scattered across multiple binaries :/

**Comment by riannucci** [comment]
Ah... hm... I don't know of a way to have goconvey's runner serialize multiple binaries, as far as I know :/

Maybe for now you could modularize the actual bulk of the testing code, and then just import it all into one test binary? Then later when they can operate in parallel, restore the independent binaries? A bit of a hack, but that would probably be the most direct way to control the parallelism.
